Sources/Brave/Frontend/Browser/Tabs/TabTray/TabTrayController.swift

```diff
@@ -793,6 +793,19 @@ class TabTrayController: AuthenticationController {
   }
 }
 
+// MARK: InterfaceOrientation
+
+extension TabTrayController {
+  
+  override var preferredInterfaceOrientationForPresentation: UIInterfaceOrientation {
```

Comment:
Pr works fine without this one afaict.

Reply:
Let me test and will remove after that.

Reply:
Fixed / Removed at https://github.com/brave/brave-ios/pull/8485/commits/8ac94bc1e30e13dfb6d42a77619106ce97830561